README.md

```diff
@@ -78,6 +78,7 @@ Initialize sdk in method `onCreate()`.
 protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     FacebookSdk.sdkInitialize(getApplicationContext());
```

Comment:
Remove call to `FacebookSdk.sdkInitialize(getApplicationContext());` because it's duplicate with the one in Application's `onCreate`